src/common/components/hosts/tableUtils.tsx

```diff
@@ -83,6 +83,7 @@ export const hostnameColumn = (
 
 export const roleColumn = (
   t: TFunction,
+  clusterKind: Cluster['kind'],
```

Comment:
there is couple of places all over the source code, where the new `clusterKind` is not passed

Reply:
In the end, I just switched the position on the parameter, made it optional and changed the condition in `getHostRole`. Things should be working now even for other usages of these functions.

Reply:
@celdrake I removed the `position` parameter because it was never passed to `roleColumn`. It's optional in `RoleCell`, `RollDropdown` and further on. 

I suppose it could be removed from those as well, since we use the default position anyway.

Reply:
If that's the case, then yes, we should be cleaning-up the unused property. Otherwise there are left-overs and in the future it's harder to track what's being used and what isn't